lib/Plack/MIME.pm

```diff
@@ -148,6 +148,7 @@ our $MIME_TYPES = {
     ".text"    => "text/plain",
     ".tif"     => "image/tiff",
     ".tiff"    => "image/tiff",
+    ".ttf"     => "font/ttf",
```

Comment:
doesn't seem to be alphabetically sorted?

Reply:
Oops. Fixed.